pkg/watchdog/watchdog.go

```diff
@@ -116,7 +120,7 @@ func WatchdogFileIsExpired(timeoutSeconds int, baseDir string, vmi *v1.VirtualMa
 		return false, err
 	}
 
-	now := time.Now().UTC().Unix()
+	now := timeNow.UTC().Unix()
```

Comment:
I think on the long run, we should mock the time properly and introduce interfaces.